hw/ip/aes/dv/aes_masking_sim_cfg.hjson

```diff
@@ -11,7 +11,7 @@
   import_cfgs: ["{proj_root}/hw/ip/aes/dv/aes_base_sim_cfg.hjson"]
 
   // Coverage exclusion
-  xcelium_cov_refine_files: ["{proj_root}/hw/ip/aes/dv/cov/aes_UNR.vRefine"]
+  xcelium_cov_refine_files: ["{proj_root}/hw/ip/aes/dv/cov/refines/aes_UNR.vRefine"]
```

Comment:
Thanks for catching this! I just checked that this also increases coverage numbers for the masked implementation as expected:
|  Score  |  Block  |  Branch  |  Statement  |  Expression  |  Toggle  |   Fsm   |  Assertion  |  CoverGroup  |
|:-------:|:-------:|:--------:|:-----------:|:------------:|:--------:|:-------:|:-----------:|:------------:|
| 96.10  | 95.83 | 90.86 | 98.35     | 92.98      | 96.25  | 97.78 | 98.67     | 92.87      | 
| 96.76 | 96.52 | 91.98  |   98.62   |   94.66    | 96.33  | 97.90 |   98.39   |   91.85    |